src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -553,19 +556,6 @@ export default class AirportModel {
      */
     getWind = () => {
```

Comment:
see note below on where this is used, but this could probably be removed entirely